src/Application/DatabaseManager/DatabaseManager.cc

```diff
@@ -78,7 +78,7 @@ DatabaseManager::DatabaseManager( const DatabaseManager& src ) :
     // Copy the database content from the source
     sqlite3_backup* db_backup_obj = sqlite3_backup_init( this->private_->database_,
       "main", src.private_->database_, "main" );
-    if ( db_backup_obj == nullptr )
```

Comment:
I think that `DatabaseManager` is pure C++ file, so leaving `nullptr` in here is fine. I will edit this commit.

Reply:
Anything that has to do with SQL requires NULL and not nullptr. They need to stay NULL or there will be SQL errors.

Reply:
I just tried building and running. There are no SQL errors now like there were before.